message DeviceInfo
{
    required string imei = 1;
    required string platform = 2;
    required string device_type = 3;
    required string device_id = 4;
}
